scripts/data/gen-nodetime/src/relayer/index.ts

```diff
@@ -2,7 +2,7 @@ import run from "./jsonrpc";
 
 import Relayer from "./lib/relayer";
 
-const relayer = new Relayer();
+const relayer = new Relayer(parseInt(process.argv[2]));
```

Comment:
If log levels are defined as constants, maybe:

```suggestion
const logLevel = parseInt(process.argv[2])
const relayer = new Relayer(isNaN(logLevel) ? LEVEL_INFO : logLevel);
```